### PR TITLE
Delete non empty namespace test

### PR DESCRIPTION
**Pull request by ShaiahWren**
No-Issue, fills in another test from. https://github.com/himdel/ansible-hub-ui/issues/1 in the `Collections > Namespaces` department: "can only delete [namespace] when empty".

### CODE REVIEW

**Comment by himdel** [comment]
you'd need `cy.settings({ GALAXY_REQUIRE_CONTENT_APPROVAL: true })` to enable approval if you need it.

(Though, sounds like here you'd just need to wait until the import is successful since you don't actually need manual approval to test deleting namespaces.)

File: test/cypress/integration/namespace_delete.js [hunk 1]
```diff
@@ -21,4 +22,53 @@ describe('Delete a namespace', () => {
...
+    ).as('reload');
+    cy.galaxykit('-i namespace create', 'ansible');
+    cy.menuGo('Collections > Namespaces');
+    cy.wait('@reload');
+
+    cy.get('a[href*="ui/repo/published/ansible"]').click();
+
+    //upload a collection
+
+    cy.contains('Upload collection').click();
```
Comment (brumik) [inline test/cypress/integration/namespace_delete.js:40]: Why not upload the collection trough galaxykit? I thought we have separate tests for testing upload collection.

**Review by brumik — COMMENTED** [review]
Martin: `cy.settings({ GALAXY_REQUIRE_CONTENT_APPROVAL: true })` would not work, since we are overriding in the env. The PR #2069 tries to fix this.

Shaiah: Wouldn't be easier to use galaxykit at the `before` and test on existing data? So example
```js
before(() => {
  cy.deleteNamespacesAndCollections();
  galaxikit('namespace create test_delete_success');
  galaxikit('namespace create test_delete_non_empty');
  galaxikit('collection upload test_delete_non_empy coll1');
});

after(() => { cy.deleteNamespacesAndCollections(); });
```

Also `deleteNamespacesAndCollections()` is a bit slow, so using the above pattern would maybe make it more performant too.

**Review by himdel — APPROVED** [review]

**Comment by ShaiahWren** [comment]
pr closed.

**Pull request merged**